server.js

```diff
@@ -2040,6 +2040,135 @@ cache(function(data, match, sendBadge, request) {
   });
 }));
 
+// GitHub pullrequests integration.
+camp.route(/^\/github\/issues-pr(-raw)?\/([^\/]+)\/([^\/]+)\.(svg|png|gif|jpg|json)$/,
```

Comment:
Factor in all you can, if you have justification for what doesn't get factored in, I'll merge it.